packages/create-block/lib/scaffold.js

```diff
@@ -48,11 +50,12 @@ module.exports = async (
 	namespace = namespace.toLowerCase();
 
 	info( '' );
-	info( `Creating a new WordPress plugin in the "${ slug }" directory.` );
+	info( getMessage( 'create', plugin, { slug } ) );
```

Comment:
I find the code more complex and harder to update with the new helper method introduced.
This could be as simple as:

```js
info(
    plugin
        ? `Creating a new WordPress plugin in the "${ slug }" directory.`
        : `Creating a new block in the "${ slug }" directory.`
);
```

I don't see anything wrong with branching the code in so many places for now.

Reply:
Agreed. I was trying to simplify, but looks like I over-engineered 😄.